nixos/modules/services/web-apps/plausible.nix

```diff
@@ -91,6 +96,17 @@ in {
           framework docs</link>.
         '';
       };
+      listenAddress = mkOption {
+        default = "127.0.0.1";
+        type = types.str;
+        description = ''
+          The IP address on which the server is listening.
+
+          When changing listen IPs, also consider
+          <option>services.plausible.erlang.vmListenAddress</option> and
```

Comment:
I'd strongly prefer to use `<xlink linkend="opt-services.plausible.etc"/>` here as this makes links clickable :)

Reply:
> I'd strongly prefer to use `<xlink linkend="opt-services.plausible.etc"/>` here as this makes links clickable :)

You mean `<xref linked` I think. Good idea, I forgot about it. Done.